tests/sql_generators/usage_reporting/expected/moz-fx-data-shared-prod/firefox_desktop_derived/usage_reporting_clients_first_seen_v1/schema.yaml

```diff
@@ -9,5 +9,75 @@ fields:
 - mode: NULLABLE
   name: usage_profile_id
   type: STRING
+  description:
```

Comment:
description missing again (other files might need checking too)

Reply:
added a description for `usage_profile_id` to both `first_seen` and `daily` templates